python/cli/process/process.py

```diff
@@ -591,6 +598,10 @@ def detect_device_preset(input_dir):
         print(f"val-dataset-json-path: 'data/{name}/val.json'", flush=True)
     elif args.format == 'nerfstudio':
         print(f'output written to {args.output}', flush=True)
+    elif args.format == 'ply':
+        print(f'output written to {args.output}/pointCloud.ply', flush=True)
```

Comment:
It would be great if you could actually give the ouput file path as `--output /foo/bar/my_point_cloud.ply`. The script could auto-detect the format from the extension (no extension would be a folder and then the `--format` could be used to define what kind of data the folder contains, e.g., `nerfstudio` by default). The tmp folder would need to be moved elsewhere (now output/something and not a default Python tempdir), but that could be a good change anyway